base/iterators.jl

```diff
@@ -240,6 +240,32 @@ end
 iteratorsize{I1,I2}(::Type{Zip{I1,I2}}) = zip_iteratorsize(iteratorsize(I1),iteratorsize(I2))
 iteratoreltype{I1,I2}(::Type{Zip{I1,I2}}) = and_iteratoreltype(iteratoreltype(I1),iteratoreltype(I2))
 
+# unzip
+```jldoctest
+
+For iterable objects of the same length, returns iterable sets, where the ith set contains
+the ith component of each input iterable object.
+
+Note that the inverse of [`unzip`](@ref) is [`zip`](@ref).
```

Comment:
I just meant the `@ref` part of `unzip` here, no need to link to itself. (And the docs for `zip` probably shouldn't have a `@ref` for itself either.)